### PR TITLE
@mzikherman: Restores marketing signup modal for mobile

### PR DESCRIPTION
**Pull request by xtina-starr**
Closes: https://github.com/artsy/collector-experience/issues/293

Looks like we lost the marketing signup modals when we merged force and microgravity. This PR restores that as well as removes the photo credit line.

__note__: the environment variables should be set in staging and production before this gets merged. 

screen shot 2017-06-01 at 5 15 37 pm

### CODE REVIEW

File: lib/setup.js [hunk 1]
```diff
@@ -198,6 +199,9 @@ export default function (app) {
   app.use(unsupportedBrowserCheck)
   app.use(splitTestMiddleware)
 
+  // Sets up mobile marketing signup modal
+  app.use(marketingSignupModal)
```
Comment (xtina-starr) [inline lib/setup.js]: This what was missing in order to setup the marketing signup model for mobile. Is there a better place to put this?

File: lib/setup_sharify.js [hunk 2]
```diff
@@ -62,6 +62,7 @@ sharify.data = _.extend(_.pick(config,
   'MARKETING_SIGNUP_MODALS',
   'MAX_POLLS_FOR_MAX_BIDS',
   'METAPHYSICS_ENDPOINT',
+  'MOBILE_MARKETING_SIGNUP_MODALS',
```
Comment (xtina-starr) [inline lib/setup_sharify.js]: I changed name of the constant so that it didn't conflict with the desktop version.

File: mobile/config.coffee [hunk 3]
```diff
@@ -33,7 +33,7 @@ module.exports =
   IMAGE_PROXY: 'GEMINI'
   MAILCHIMP_AUCTION_LIST_ID: 'b7b9959ee0'
   MAILCHIMP_KEY: null
-  MARKETING_SIGNUP_MODALS: '[{"slug":"ca1","copy":"An art collection for every budget","image":"http://files.artsy.net/images/small-modal-interior-a.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca2","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/small-modal-interior-a.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca3","copy":"An art collection for every budget","image":"http://files.artsy.net/images/small-modal-interior-b.jpg","photoCredit":"Photo by Emily Johnson"},{"slug":"ca4","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/small-modal-interior-b.jpg","photoCredit":"Photo by Emily Johnson"}]'
+  MOBILE_MARKETING_SIGNUP_MODALS: '[{"slug":"ca1","copy":"An art collection for every budget","image":"http://files.artsy.net/images/modal-collect-art.jpg"},{"slug":"ca2","copy":"Buy art from the best galleries and auction houses","image":"http://files.artsy.net/images/modal-collect-art.jpg"}]'
```
Comment (xtina-starr) [inline mobile/config.coffee]: I changed the name of the constant so that it didn't conflict with the desktop version.

**Comment by xtina-starr** [comment]
I'm not sure, but this may be a good candidate to pull down for simple testing since I'm adding to the middleware. What do you guys think? @mzikherman @yuki24

File: mobile/config.coffee (hunk 3 above)
Comment (mzikherman) [inline mobile/config.coffee]: 👍

File: lib/setup.js (hunk 1 above)
Comment (mzikherman) [inline lib/setup.js]: Seems good to me!

**Comment by mzikherman** [comment]
> I'm not sure, but this may be a good candidate to pull down for simple testing since I'm adding to the middleware. What do you guys think? 

Possibly, but I'm not sure. I think a good or better candidate would be for a newer or bigger feature, esp. with some heavier UI interaction. Something like this, which seems more like cleanup/oversight, I personally would be comfortable _not_ checking out myself once a developer has deemed it ready and opened a PR.

We can also always use staging 😄 to do further testing, we just have to be cognizant to not clog up the deploy pipeline too much.

**Comment by xtina-starr** [comment]
I've added the environment variables to staging and production so this should be ready to go.

**Comment by mzikherman** [comment]
Aw yea, love the pre-deploy migration!

**Pull request merged**